EIPS/eip-5450.md

```diff
@@ -13,67 +13,127 @@ requires: 3540, 3670, 4200, 4750
 
 ## Abstract
 
-Introduce extended validation of code sections to guarantee that stack underflow cannot happen during execution of validated contracts.
+Introduce extended validation of EOF code sections to guarantee that stack underflow and overflow cannot happen during execution of validated contracts.
 
 ## Motivation
 
-Currently existing EVM implementations perform a number of validity checks for each executed instruction, such as check for stack overflow/underflow, sufficient gas, etc. This change aims to minimize the number of such checks required at run-time, by verifying at deploy-time that no exceptional conditions can happen, and preventing to deploy the code where it could happen.
+The current EVM performs a number of validity checks for each executed instruction, such as checking
+for instruction being defined, stack overflow and underflow, and enough amount of gas remaining.
+
+This EIP minimizes the number of such checks required at run-time
+by verifying that no exceptional conditions can happen
+and preventing the execution and deployment of any invalid code.
+
+The operand stack validation provides several benefits:
+
+- removes run-time stack underflow check for all instructions,
+- removes run-time stack overflow check for all instruction except `CALLF`,
+- ensures that an execution terminates with one of the terminating instructions,
```

Comment:
Is this still relevant?

Reply:
I believe it is, but this doesn't meen final byte is necessary a terminating instruction (can be unconditional jump too)